Neos.Flow/Classes/Mvc/Controller/ActionController.php

```diff
@@ -200,6 +210,9 @@ public function processRequest(ActionRequest $request, ActionResponse $response)
         $this->mvcPropertyMappingConfigurationService->initializePropertyMappingConfigurationFromRequest($this->request, $this->arguments);
 
         $this->mapRequestArgumentsToControllerArguments();
+        if ($this->enableDynamicTypeValidation === true) {
```

Comment:
and
```suggestion
        if ($this->enableDynamicTypeValidation) {
```
?

(I can totally live with both ofc)